File: gs-uploading-files-main/initial/src/main/java/com/example/uploadingfiles/UploadingFilesApplication.java
package com.example.uploadingfiles;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class UploadingFilesApplication {

  public static void main(String[] args) {
    SpringApplication.run(UploadingFilesApplication.class, args);
  }

}

File: gs-uploading-files-main/initial-kotlin/src/main/kotlin/com/example/uploadingfiles/UploadingFilesApplication.kt
package com.example.uploadingfiles

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class UploadingFilesApplication

fun main(args: Array<String>) {
	runApplication<UploadingFilesApplication>(*args)
}